Assets/BossRoom/Scripts/Shared/Net/GameNetPortal.cs

```diff
@@ -248,6 +255,47 @@ public void StartRelayHost(string roomName)
             NetManager.StartHost();
         }
 
+        public async void StartUnityRelayHost()
+        {
+            var chosenTransport  = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().UnityRelayTransport;
+            NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
+
+            switch (chosenTransport)
+            {
+                case UnityTransport utp:
+                    Debug.Log("Setting up UTP relay host");
+                    // TODO: This needs to be removed ?
+                   // Unity.Services.Relay.RelayService.Configuration.BasePath = "https://relay-allocations-stg.services.api.unity.com";
+                    await UnityServices.InitializeAsync();
+                    if (!AuthenticationService.Instance.IsSignedIn)
+                    {
+                        await AuthenticationService.Instance.SignInAnonymouslyAsync();
+                        var playerId = AuthenticationService.Instance.PlayerId;
+                        Debug.Log(playerId);
+                        // we now need to get the joinCode?
+                        var serverRelayUtilityTask = RelayUtility.AllocateRelayServerAndGetJoinCode(10);

```

Comment:
magic number 10?

Reply:
It happens to be the max number of players the relay supports. Which is not really documented on the relay side :P

Reply:
ok. one more reason to document it here. We could have a const defined for this.